x-pack/solutions/observability/plugins/observability_ai_assistant_app/scripts/evaluation/scenarios/elasticsearch/index.spec.ts

```diff
@@ -136,6 +137,265 @@ describe('Elasticsearch function', () => {
+    const TEST_DATAFEED_ID = `datafeed-${TEST_JOB_ID}`;
+    const SERVICE_TEST_NAME = 'web-api';
+
+    before(async () => {
+      await esClient.indices.create({
+        index: TEST_INDEX,
+        mappings: {
+          properties: {
+            '@timestamp': { type: 'date' },
+            response_time: { type: 'float' },
+            service: { type: 'keyword' },
+          },
+        },
+      });
+
+      const now = new Date();
+      const bulkBody: any[] = [];
+
+      // Normal data points
+      for (let i = 0; i < 45; i++) {
+        bulkBody.push({ index: { _index: TEST_INDEX } });
+        bulkBody.push({
+          '@timestamp': new Date(now.getTime() - i * 60000).toISOString(),
+          response_time: 50 + Math.random() * 20, // Normal: 50-70ms
+          service: SERVICE_TEST_NAME,
+        });
+      }
+
+      // Anomalous data points
+      for (let i = 0; i < 5; i++) {
+        bulkBody.push({ index: { _index: TEST_INDEX } });
+        bulkBody.push({
+          '@timestamp': new Date(now.getTime() - (i + 10) * 60000).toISOString(),
+          response_time: 300 + Math.random() * 100, // Anomalous: 300-400ms
+          service: SERVICE_TEST_NAME,
+        });
+      }
+
+      await esClient.bulk({ refresh: true, body: bulkBody });
+
+      // Create ML job
+      const mlJobRequest: MlPutJobRequest = {
+        job_id: TEST_JOB_ID,
+        description: 'Detect anomalies in average response_time',
+        analysis_config: {
+          bucket_span: '5m',
+          detectors: [
+            {
+              function: 'mean',
+              field_name: 'response_time',
+            },
+          ],
+        },
+        data_description: {
+          time_field: '@timestamp',
+        },
+        datafeed_config: {
+          datafeed_id: TEST_DATAFEED_ID,
+          indices: [TEST_INDEX],
+          query: { match_all: {} },
+        },
+      };
+
+      await esClient.ml.putJob(mlJobRequest);
+
+      await esClient.ml.openJob({ job_id: TEST_JOB_ID });
+
+      // Start datafeed and wait for processing
+      await esClient.ml.startDatafeed({
+        datafeed_id: TEST_DATAFEED_ID,
+        start: 'now-1h',
+      });
+
+      // Wait for job to process data
+      await new Promise((resolve) => setTimeout(resolve, 10000));
+    });
+
+    after(async () => {
+      try {
+        await esClient.ml.stopDatafeed({ datafeed_id: TEST_DATAFEED_ID, force: true });
+        await esClient.ml.closeJob({ job_id: TEST_JOB_ID, force: true });
+        await esClient.ml.deleteDatafeed({ datafeed_id: TEST_DATAFEED_ID });
+        await esClient.ml.deleteJob({ job_id: TEST_JOB_ID });
+      } catch (error) {
+        logger.info('Cleanup warning:', error);
+      }
+      await esClient.indices.delete({ index: TEST_INDEX, ignore_unavailable: true });
+    });
+
+    it('returns a list of ML jobs', async () => {
+      const conversation = await chatClient.complete({
+        messages: 'List all Machine Learning (ML) jobs',
```

Comment:
It's good to keep the prompt closer to how a user would type it.

```suggestion
        messages: 'List all machine learning jobs',
```